flink-yarn/src/test/java/org/apache/flink/yarn/UtilsTest.java

```diff
@@ -58,4 +87,84 @@ public void testDeleteApplicationFiles() throws Exception {
 			assertThat(files.count(), equalTo(0L));
 		}
 	}
+
+	@Test
+	public void testCreateTaskExecutorContext() throws Exception {
```

Comment:
This is very much involved, I was wondering if it's better to have a separate util method to just simply test this specific piece of removing AM_RM_TOKEN from the credential factory. and/or move it to `flink-yarn-test` package to have a more comprehensive testing?
e.g.
```
		Assert.assertThat(
			"AMRMToken", taskManagerRunsWithKerberos, Matchers.is(false));
```
in `YARNSessionFIFOSecuredITCase` ??

Reply:
Yeah, I think it's sort of complicated as a unit test and does not have good separation of concerns. Actually, the test is extended from a test in legacy mode that only verifies the local resources of TaskManagers. It's a good idea to refactor it in a util method and move it to `flink-yarn-tests`.

Reply:
@walterddr I've moved the test to `flink-yarn-tests` and added an assertion to verify container tokens in `YARNSessionFIFOSecuredITCase`. PTAL :)